### PR TITLE
Removed unused code.

### PR DESCRIPTION
**Pull request by rfreytag**
Deleted unused code as discussed in a prior pull request (now closed by me).

### CODE REVIEW

File: src/Samples/NewWindow.Core/AutoOpen.fs [hunk 1]
```diff
@@ -30,7 +30,3 @@ module InOutModule =
     let cata f g = function
       | InOut.In  msg -> msg |> f
       | InOut.Out msg -> msg |> g
-
```
Comment (LyndonGingerich) [inline src/Samples/NewWindow.Core/AutoOpen.fs:33]: Tyson might want a closing newline.

**Review by LyndonGingerich — APPROVED** [review]

File: src/Samples/NewWindow.Core/AutoOpen.fs [hunk 2]
```diff
@@ -30,7 +30,3 @@ module InOutModule =
     let cata f g = function
       | InOut.In  msg -> msg |> f
       | InOut.Out msg -> msg |> g
```
Comment (LyndonGingerich) [inline src/Samples/NewWindow.Core/AutoOpen.fs:32]:
```suggestion
      | InOut.Out msg -> msg |> g

```
Tyson might want a closing newline.

Comment (LyndonGingerich) [reply src/Samples/NewWindow.Core/AutoOpen.fs:32]: It's generally a good idea anyway.

Comment (rfreytag) [reply src/Samples/NewWindow.Core/AutoOpen.fs:32]: Doh!  I thought I had.  Especially important in languages like F# where newlines fill the role `;`s in other languages.

**Comment by TysonMN** [comment]
I tried updating the branch in the GitHub app. I forgot that it always uses a merge commit. I don't want there to be a merge commit. I should have used the GitHub website from my phone.

Can you force push to the branch in this PR just the first commit rebased on `master`?

File: src/Samples/NewWindow.Core/AutoOpen.fs (hunk 2 above)
Comment (LyndonGingerich) [reply src/Samples/NewWindow.Core/AutoOpen.fs:32]: This PR still seems to remove the closing newline.

**Comment by rfreytag** [comment]
@TysonMN Sorry, not experienced with `git rebase`.  I have avoided it in my team's development due potential confusion if abused.

This is the first project I have wanted to contribute to.  I am concerned I will make matters worse. 

Am I correct that one should pull, change, and finally issue a push (issue a pull request) a commit to meet your preferences?

**Comment by LyndonGingerich** [comment]
@rfreytag Tyson has accidentally pushed a merge commit, which he does not want. You don't need to rebase, just force push to get rid of the merge commit. Don't pull; just force push. Then Tyson can rebase if he wants.

**Comment by LyndonGingerich** [comment]
@rfreytag Or, if you want, join me at https://meet.jit.si/PartTimeSwimmingsRainLargely and I'll teach you to rebase.

**Comment by TysonMN** [comment]
It's ok. I can do it.  I will also make a video of my actions to share with you.

**Comment by TysonMN** [comment]
Here is a video:
1. Remove merge commit (by doing a hard reset to the previous commit)
2. Rebase on `master`
3. Force push

I am using [GitKraken ](https://www.gitkraken.com/)there, which I highly recommend.

2022-09-16_22-24-27_717_gitkraken

**Pull request merged**